assignment.md

```diff
@@ -59,22 +57,30 @@ Git includes three commands that you can use to review, update, and publish your
 Find more information about these commands in the sections to follow.
 
 ### Retrieving code updates (`fetch`)
-The `fetch` command first checks your local branch to determine whether an associated <!-- tracking? remote? --> branch exists. If the tracking branch exists, `fetch` will retrieve the changes from the <!-- tracking? remote? --> branch and pulls those changes into the <!-- tracking? remote? --> branch.
+The `fetch` command first checks your local branch to determine whether an associated tracking branch exists remotely. If the tracking branch exists, `fetch` retrieves the changes and history from the tracking branch and pulls those changes into the <!-- tracking? remote? --> branch.
 
 The `fetch` command does not update your local branch.
 
-<!-- EAJ QUESTION: The text that was originally here was confusing and a jumble of terminology. -->
+<!-- EAJ QUESTIONS:
+    - The text that was originally here was confusing and a jumble of terminology. I am primarily trying to get straight what is updated and where. I understand that it downloads a branch from a remote repository (either the origin or an upstream branch), including the history of each asset. What I don't understand is when we say "retrieve" or "download," where is this downloaded to? It says my local branch isn't updated, but how can I review it locally if my local branch isn't updated? Is it updated on a remote repository?
+    - Does the user need to run fetch before they pull or push? How often?
+-->
+
+### Retrieving changes and merging them to your local repository (`pull`)
+The `pull` command fetches changes from the tracking branch or repository, then applies those changes to your local repository so that you have the latest copy.
```

Comment:
Is there more information that we can pull in here from other resources? The document won't feel right with such short sections.

- Consider providing background for what else it does.
- Why is pulling important? Is there a situation in which you should pull updates?
- What are some external resources?